testutil/network/network.go

```diff
@@ -46,7 +46,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
 	_ "github.com/cosmos/cosmos-sdk/x/auth"
-	_ "github.com/cosmos/cosmos-sdk/x/auth/tx/module"
```

Comment:
not really relevant here, but I think we should move this out of auth in the future. It's weird that all modules rely on auth because of this.

Reply:
Yes, see my comment on Slack :)